courageous_comets/redis/messages.py

```diff
@@ -159,7 +159,13 @@ async def get_message_sentiment(
     if not data:
         return None
     return models.SentimentResult.model_validate(
-        dict(zip(fields, map(float, data), strict=True)),
+        dict(
```

Comment:
This error happened because you probably had data in the db while the models were being changed. For instance, at one point, we weren't storing sentiment data. Then, we added those. Subsequently, we didn't have tokens. 

Given the model is pretty much set now, all fields will have values and this won't be necessary. You might want to flush your development db.

Reply:
This handles an error that happens if you try to get the analysis for a message that's not in the DB. For example if the bot was not running at the time. Figured if there was no value, I'd just take 0 as a default.

Reply:
oh, okay.